test/org.test.js

```diff
@@ -0,0 +1,88 @@
+const app = require('../src/index.js');
+const chai = require("chai");
+const { before, describe, it } = require('mocha');
+const chaiHttp = require("chai-http");
+const { expect } = chai;
+chai.use(chaiHttp);
+
+describe('organization', () => {
+    let token;
+
+    before('get admin api token', function(done) {
+        chai.request(app)
+            .put('/auth')
+            .send('email=admin')
+            .send('password=admin')
+            .set('Accept', 'application/json')
+            .end((err, res) => {
+                expect(res).to.have.status(200);
+                expect(res.body.token).lengthOf(828);
+                token = res.body.token;
+                done();
+            })
```

Comment:
Why no semicolons at the end of the function calls?

Reply:
(This is an insignificant question, more about linting than actual problems with your code 😂 Don't worry about it)

Reply:
I haven't set up prettier or any linting yet and have recently been programming in Go a lot which doesn't use semi-colons haha. When I set up a linter it'll fix the formatting details